app/components/icons/TouchIdIcon.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react'
+import Svg, { G, Path, SvgProps } from 'react-native-svg'
+
+export function TouchIdIcon (props: SvgProps): JSX.Element {
+  return (
+    <Svg width='49' height='50' viewBox='0 0 49 50' {...props}>
+      <G id='Onboarding' stroke='none' stroke-width='1' fill='none' fill-rule='evenodd' stroke-linecap='round'>
```

Comment:
same comments here from Settings PR (regarding camelCase)

Reply:
> For the TransactionAuthorization page, can we remove/disable cancel button once it's Signing? Also, when cancelling the Biometeric authentication on any transaction, it shouldn't trigger a validation. We should let users input Pin Setup.

The LocalAuthentication has quite limited flow control allowed.
Current solution:
- no fallback to device pin allowed when actual decryption needed using in-app pin
- thus user CANNOT fallback to device pin, (1) limit guess attempt, (2) the OS biometric prompt FALLBACK option disabled, once canceled, it will come back to here which is in-app pin input UI
- Privacy Lock will different implementation, never use in-app pin